vl-infotext.src.js

```diff
@@ -86,25 +86,33 @@ export class VlInfotext extends NativeVlElement(HTMLDivElement) {
         }
     }
 
-    __observeElement(element, callback) {
-        const observer = new ResizeObserver(callback);
-        observer.observe(element);
-		return observer;
+    __observeChildElement(callback) {
+        const observer = new MutationObserver(callback);
+        observer.observe(this.firstElementChild, { childList: true, subtree: true });
+        return observer;
     }
 
-    __observeValueElement(callback) {
-        return this.__observeElement(this.__valueElement, callback);
-	}
-
-    __observeTextElement(callback) {
-        return this.__observeElement(this.__textElement, callback);
-	}
+    __observeChildElementResize(callback) {
+        const observer = new ResizeObserver(callback);
+        observer.observe(this.firstElementChild);
+        return observer;
+    }
 
-    __processValueElement() {
-        vl.infotext.dress(this.__valueElement);
+    __processChildElementChange(records) {
+        records.forEach(record => {
+            const isChildListType = record.type == 'childList';
+            const hasOneAddedNode = record.addedNodes && record.addedNodes.length == 1;
+            const hasOneRemovedNode = record.removedNodes && record.removedNodes.length == 1;
+            const isTextContentChanged = hasOneAddedNode && hasOneRemovedNode && record.addedNodes[0].textContent != record.removedNodes[0].textContent;
+            if (isChildListType && hasOneAddedNode && hasOneRemovedNode && isTextContentChanged) {
```

Comment:
Ter info voor andere reviewers: het was me onduidelijk of de observer altijd een removed en added node ging hebben in dezelfde record. Bijvoorbeeld in het geval dat men een node removed en dat de DOM toevallig net nadien rerendered wordt door de browser, en dan pas een node added werd. 
@coemans heeft dat opgezocht en blijkbaar gebeurt er een aanpassing van de tekst in de dress van vl-infotext, ipv. een node remove&add. Volgens wat @coemans gevonden had zorgt een aanpassing van de tekst blijkbaar altijd voor een removedNode en een addedNode in de observer record.